microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -594,6 +594,439 @@
         select.prop("disabled", false);
     }
 
+    function drawBoxPlots(top5) {
+        let mockData = top5.map(d => {
+            const relativeAbundance = parseFloat(d.relativeAbundance);
+
+            let maxRange;
+            if (relativeAbundance > 50) {
+                maxRange = 100;
+            } else if (relativeAbundance < 50 && relativeAbundance > 5) {
+                maxRange = 50;
+            } else {
+                maxRange = 5;
+            }
+
+            const minVal = 0;
+            const maxVal = maxRange;
+            const q1 = relativeAbundance * 0.25;
+            const median = relativeAbundance * 0.5;
+            const q3 = relativeAbundance * 0.75;
+
+            return {
+                genus: d.Genus || d.genus || "Unknown",
+                relativeAbundance: relativeAbundance,
+                stats: { min: minVal, q1: q1, median: median, q3: q3, max: maxVal }
+            };
+        });
+
+        const columns = 2;
+        const panelWidth = 600;
+        const panelHeight = 240;
+        const spacingX = 40;
+        const spacingY = 70;
+
+        const totalWidth = 1280;
+        const totalBoxWidth = columns * panelWidth + (columns - 1) * spacingX;
+        const remainingWidth = totalWidth - totalBoxWidth;
+
+        const rows = Math.ceil(mockData.length / columns);
+
+        const margin = { top: 20, right: 0, bottom: 60, left: -20 };
+
+        const totalHeight = rows * panelHeight + (rows - 1) * spacingY + margin.top + margin.bottom;
+
+        d3.select("#boxplot-svg").selectAll("svg").remove();
+
+        const svg = d3.select("#boxplot-svg")
+            .append("svg")
+            .attr("width", totalWidth)
+            .attr("height", totalHeight);
```

Comment:
Drawing this all as one big SVG is antithetical to the responsive design approach that we've tried to take with the rest of the interface. It renders fine given sufficient screen real estate, but on a tablet or phone screen it fails to render properly.